base/Dockerfile

```diff
@@ -21,28 +21,64 @@ LABEL License="MIT License"
 # Start s6
 ENTRYPOINT [ "/init" ]
 
+ENV \
+  # renovate: datasource=repology depName=alpine_3_20/bash
+  BASH_VERSION=5.2.26-r0 \
+  # renovate: datasource=repology depName=alpine_3_20/curl
```

Comment:
We'll have to remember to update `alpine_3_20` when we change the base `alpine` layer, though I suspect builds will fail if we reference older versions of packages. So I doubt we'll run into a discrepancy, since builds will fail.